### PR TITLE
process CvMat as well

### PR DESCRIPTION
**Pull request by kameamea**
Hi, I had to debug some internal opencv code where they use CvMat. With those changes one can process them as well.

### CODE REVIEW

File: resources/giwscripts/giwtype.py [hunk 1]
```diff
@@ -30,10 +30,16 @@ def get_buffer_metadata(obj_name, picked_obj, debugger_bridge):
...
     width = int(picked_obj['cols'])
     height = int(picked_obj['rows'])
+    if hasattr(picked_obj,'flags'):
     flags = int(picked_obj['flags'])
+    else:
+        flags = int(picked_obj['type'])
 
     channels = ((((flags) & CV_MAT_CN_MASK) >> CV_CN_SHIFT) + 1)
+    if hasattr(picked_obj['step'],'buf'):
     row_stride = int(int(picked_obj['step']['buf'][0])/channels)
```
Comment (csantosbh) [inline resources/giwscripts/giwtype.py]: I'm not sure github is correctly showing the diff, but shouldn't this line be indented to its if block?

**Comment by csantosbh** [comment]
Great, thanks! I made one comment regarding the diff, but other than that, it looks great!

File: resources/giwscripts/giwtype.py (hunk 1 above)
Comment (kameamea) [reply resources/giwscripts/giwtype.py]: of course you are right, line 40 has to be indented, I must have made some mistake in merging this solution to the github branch.

**Comment by kameamea** [comment]
I broke handling of cv::Mat, but with the latest 733b06d it should now really both work

**Comment by csantosbh** [comment]
Hey, thanks for checking that out! I didn't have a time to test it yet, but if you are having any problems implementing the giwtypes for one particular type, note that you can do so by implementing a new separate class. This way, you won't affect the functionality of the other type, and both types will work automatically :)

**Comment by csantosbh** [comment]
Looks good, merging into master. Thanks!

**Pull request merged**